src/Publisher/Amazon/SNS/Application/PlatformEndpoint/Publish.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ *  The MIT License (MIT)
```

Comment:
Then we'll still use an opensource license: 
https://opensource.org/licenses/BSD-3-Clause